profile/latest/advisor/management/advisor/models.go

```diff
@@ -0,0 +1,40 @@
+package advisor
+
+import (
+	 original "github.com/Azure/azure-sdk-for-go/service/advisor/management/2017-04-19/advisor"
```

Comment:
I am not a big fan of duplicating `advisor` in the import path, however, I think I understand why you did it that way.